_data/students.yml

```diff
@@ -546,7 +546,7 @@
   twitter: Pro_MYScrub
   bio: Can you differentiate the useful and the useless?
 
-- name: Yee Jia Shuen
+- name: Yee J. S.
```

Comment:
I guess, if need be, she shall change her name on her own :)
For the time being, kindly just add up your details so that your name is there on the website too !